server/services/diskUsageService.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * This service is not per rtorrent session, which is why it does not inherit
+ * `BaseService` nor have any use of the per user API ie. `getSerivce()`
+ */
+const EventEmitter = require('events');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const diskUsageServiceEvents = require('../constants/diskUsageServiceEvents');
+
+const diskUsage = {
+  linux: () =>
+    exec('df --block-size=1 --type=ext4 --portability | tail -n+2').then(({stdout}) =>
+      stdout
+        .trim()
+        .split('\n')
+        .map(disk => {
+          const [, /* fs */ size, used, avail /* pcent */, , target] = disk.split(/\s+/);
```

Comment:
What do the comments mean? Also do you happen to know what the values for indices 0 and 4 represent? It would be nice to define the unused variables, prefixed with underscores (e.g. `_foo`).

Reply:
the columns of df output.. that's their short names, originally from df manual pages